src/utils/api.js

```diff
@@ -24,17 +24,13 @@ class Api {
         )
     }
 
-    createNamespace(namespace, key, value) {
+    createNamespace(namespace, key, value = {}) {
         return getInstance()
             .then(d2 => {
-                console.log(d2.dataStore)
                 d2.dataStore.create(namespace).then(namespace => {
```

Comment:
I think you also need a `return` on this line. e.g.

return d2.dataStore.create(...

Reply:
hehe completely forgot about this, thanks now it's working, I'll save it and it will be ready